### DISCUSSION TITLE
Support datadog monitoring

### DISCUSSION THREAD

**Discussion by akshay0709** [jertel/elastalert2#875] [Ideas]
Hello,

We used Datadog for monitoring our infrastructure. Wanted to check the level of efforts we would need to add datadog support for this project. Currently, datadog is only supported for alerting and not monitoring. From what I understand, we would need to import datadog client and send the metrics using dogstatsd as show here: https://github.com/DataDog/datadogpy#dogstatsd. We would need a datadog wrapper similar to the existing [prometheus_wrapper](https://github.com/jertel/elastalert2/blob/master/elastalert/prometheus_wrapper.py).
I would be happy to add this feature. Please let me know if my understanding is correct and if I should proceed with this?

Regards,
Akshay

**Comment by jertel** [comment]
Hello! I expect the level of effort for the actual implementation to be relatively small, primarily since you can use the existing Prometheus Wrapper as a template. The majority of the effort will likely be spent writing unit tests and testing the solution with a running DataDog environment. If you're up for the work, and are willing to complete all of the contribution requirements, then I don't see a problem with you submitting a PR for this. Thanks for your interest in this project!
